scripts/postprocessing/coverage_analysis/coverage_Painter.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
+
+import sys, os, re, subprocess
+import math
+
+import numpy as np
+from tf.transformations import *
+
+import time
+from visualization_msgs.msg import Marker
+from visualization_msgs.msg import MarkerArray
+import rospy
+
+"""
+Show spheres/markers representing where nav_cam was when it registered spheres
+representing Map Landmarks.
+"""
+
+class Coverage_Painter:
+
+    def __init__(self):
+       print("Coverage_Painter started")
+       self.max_num_features = 0
+       self.first_bin = 0
+       self.second_bin = 0
+       self.third_bin = 0
+       self.fourth_bin = 0
+       self.fifth_bin = 0
+
+       self.counter = 0
+       self.cube_size = 0.30 #0.15 #0.30  #Meters
+       self.markerArray = MarkerArray()
+
+
+
+    def gen_marker(self, marker_id, marker_type, Point, Rot, Color, scale, text):
+       """Place a marker of a given type and color at a given location"""
+       marker = """  -
```

Comment:
Why are you writing this to a string instead of just to the ros message type? Why is it being written to a text file? Very confused